index.html

```diff
@@ -567,7 +567,7 @@ <h4 class="text-white"><strong>PSLab Android</strong></h4>
 									<a href="https://play.google.com/store/apps/details?id=org.fossasia.phimpme">
 										<img alt="Phimp.me" src="img/Phimpme_Android.jpg"class="img-responsive">
                     <div class="text-block">
-    										<h4 class="text-white"><strong>Phimp.me Android</strong></h4>
+    										<h4 class="app-name"><strong>Phimp.me Android</strong></h4>
```

Comment:
Don't change the class name just set class name to text-black it will fixed 

Try this

Reply:
ok